components/openindiana/ddu/Makefile

```diff
@@ -0,0 +1,92 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2013 Adam Stevko. All rights reserved.
+# Copyright 2014 Alexander Pyhalov. All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		ddu
+COMPONENT_SRC=		$(COMPONENT_NAME)
+COMPONENT_VERSION=	1.3.1
+COMPONENT_LICENSE=	CDDL
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
+COMPONENT_FMRI= diagnostic/ddu
+COMPONENT_CLASSIFICATION= System/Hardware
+COMPONENT_LICENSE= CDDL,BSD3
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
+
+COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+
+COMPONENT_SUMMARY=	Device Driver Utility (DDU)
+
+GIT=git
+GIT_REPO=https://github.com/pyhalov/ddu
```

Comment:
Before moving DDU under OI umbrella I want to finish some work:
- Update pci and usb ids
- Prepare tooling to auto-generate driver.db